app/view_utils/topbar_helper.rb

```diff
@@ -58,6 +58,10 @@ def topbar_props(community:, user: nil, community_locales:, path_after_locale_ch
           imageHeight: '44px',
           image: Maybe(user).image.url(:thumb).or_else(missing_profile_image_path()),
         }
+      },
+      newListingButton: {
+        text: I18n.t("homepage.index.post_new_listing"),
+        customColor: CommonStylesHelper.marketplace_colors(community)[:marketplace_color1]
```

Comment:
Good point. It seems that we have both conventions already before this, and I just copied the case style from the avatarDropdown props above. I'll add a separate task to clean these up outside this PR.